expression/builtin.go

```diff
@@ -76,6 +76,7 @@ type baseBuiltinFunc struct {
 	foldable  bool // Default value is true because many expressions are foldable.
 	tp        *types.FieldType
 	pbCode    tipb.ScalarFuncSig
+	deferred  bool
```

Comment:
“deferred” represents whether or not a function can be cached and evaluated just before query processing via PB. One of non-deterministic functions (e.g., random) can be marked as “deferred=true” if the functions could not be pushed down to TiKV. It might be a new feature to support these functions in this plan cache.

Reply:
when a function, whether it is deterministic or not, is marked as `deferred`, it can not be pushed down to TiKV ?

Reply:
Yes.